frontend/js/views/tracks-selection.js

```diff
@@ -92,20 +97,14 @@ define(
              * Constructor
              */
             initialize: function () {
-                _.bindAll(
-                    this,
-                    "show",
-                    "hide",
-                    "search"
-                );
+
+                Modal.prototype.initialize.apply(this, arguments);
```

Comment:
Comprehension questions: Should this really be called on the prototype? Wouldn't this apply the arguments to ALL `views/modal` modals? Might be me misunderstanding prototypes, but an explanation would be nice :D